src/smart-components/group/add-group/add-group-wizard.js

```diff
@@ -88,36 +78,12 @@ const AddGroupWizard = ({
 
   const  onSubmit =  async() => {
     const user_data = { ...formData, user_list: selectedUsers ? selectedUsers.map(user => ({ username: user.label })) : undefined };
-    const group = await addGroup(user_data);
-    if (formData.policy && selectedRoles && selectedRoles.length > 0) {
-      const policy_data = {
-        name: formData.policy.name,
-        description: formData.policy.description,
-        group: group.value.uuid,
-        roles: selectedRoles.map(role => role.value)
-      };
-      return postMethod ? createPolicy(policy_data).payload.then(() => postMethod()).catch(() =>
-        addNotification({
-          variant: 'danger',
-          title: `Add group`,
-          dismissable: true,
-          description: `Error creating policy`
-        })).then(() => push(closeUrl)) :
-        createPolicy(policy_data).payload.catch(() =>
-          addNotification({
-            variant: 'danger',
-            title: `Add group`,
-            dismissable: true,
-            description: `Error creating policy`
-          })).then(() => push(closeUrl));
+    await addGroup(user_data);
+    if (postMethod) {
```

Comment:
Instead of this you can add `postMethod: () => undefined` to `AddGroupWizard.defaultProps`